Formula/x11vnc.rb

```diff
@@ -50,7 +50,7 @@ def install
 --- a/x11vnc/solid.c
 +++ b/x11vnc/solid.c
 @@ -177,7 +177,7 @@ unsigned long get_pixel(char *color) {
- 
+
```

Comment:
I think your editor is automatically trimming whitespace on save (many code editors do this) and then you (somewhat carelessly) used a plain `git add` on the changed file and thus included the change in your commit. You can usually prevent such accidental changes if you use `git add -p` instead and confirm the changes one diff block at a time, skipping unwanted changes.

The easiest thing at this point is to redo your commits, omitting the unwanted whitespace change:

```
git checkout x11vnc
git reset HEAD~2
git add -p
git commit
git push --force-with-lease
```

(Make sure to take a look at the documentation to understand what each of these commands does.)